src/tests/join/join-tests.cu

```diff
@@ -560,14 +558,15 @@ typedef ::testing::Types<
 
 TYPED_TEST_CASE(JoinTest, Implementations);
 
-TYPED_TEST(JoinTest, ExampleTest)
+TYPED_TEST(JoinTest, DISABLED_DebugTest)
```

Comment:
Why is this test being disabled?

Reply:
This test is disabled because it's a test I use for debug purposes. It has small input sizes and has extensive debug printing enabled. Having it enabled by default would add excessive clutter to the unit test output. I'll add a comment to clarify this.

Reply:
OK! Thanks